grunt failed on Windows environment

grunt command failed when I run it on Windows environment.
The stack trace is as follows.

```
   3) LocalFileSystem should return a newly saved library object:

      AssertionError: expected Array [] to equal Array [ Object { fn: 'file3.js', mno: 'pqr' } ] (at length, A has 0 and B has 1)
      + expected - actual

      -[]
      +[
      +  {
      +    "fn": "file3.js"
      +    "mno": "pqr"
      +  }
      +]
      
    at Assertion.fail (node_modules\should\lib\assertion.js:92:17)
    at Assertion.value (node_modules\should\lib\assertion.js:164:19)
    at test\red\runtime\storage\localfilesystem_spec.js:592:38
    at tryCatchReject (node_modules\when\lib\makePromise.js:845:30)
    at runContinuation1 (node_modules\when\lib\makePromise.js:804:4)
    at Fulfilled.when (node_modules\when\lib\makePromise.js:592:4)
    at Pending.run (node_modules\when\lib\makePromise.js:483:13)
    at Scheduler._drain (node_modules\when\lib\Scheduler.js:62:19)
    at Scheduler.drain (node_modules\when\lib\Scheduler.js:27:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

--- a/test/red/runtime/storage/localfilesystem_spec.js
+++ b/test/red/runtime/storage/localfilesystem_spec.js
@@ -589,17 +589,19 @@ describe('LocalFileSystem', function() {
                 flows.should.eql([ 'C', { ghi: 'jkl', fn: 'file2.js' }, {fn:'flow.json'} ]);
                 var ft = path.join("B","D","file3.js");
                 localfilesystem.saveLibraryEntry('object',ft,{mno:'pqr'},"// another non meta line\n\n Hi There").then(function() {
-                    localfilesystem.getLibraryEntry('object',path.join("B","D")).then(function(flows) {
-                        flows.should.eql([ { mno: 'pqr', fn: 'file3.js' } ]);
-                        localfilesystem.getLibraryEntry('object',ft).then(function(body) {
-                            body.should.eql("// another non meta line\n\n Hi There");
-                            done();
+                    setTimeout(function() {
+                        localfilesystem.getLibraryEntry('object',path.join("B","D")).then(function(flows) {
+                            flows.should.eql([ { mno: 'pqr', fn: 'file3.js' } ]);
+                            localfilesystem.getLibraryEntry('object',ft).then(function(body) {
+                                body.should.eql("// another non meta line\n\n Hi There");
+                                done();
+                            }).otherwise(function(err) {
+                                done(err);
+                            });
                         }).otherwise(function(err) {
                             done(err);
-                        });
-                    }).otherwise(function(err) {
-                        done(err);
-                    });
+                        })}
+                        , 50);
                 }).otherwise(function(err) {
                     done(err);
                 });

